test/e2e/tests/settings/ipfs-ens-resolution.spec.js

```diff
@@ -1,190 +1,56 @@
-const {
-  defaultGanacheOptions,
-  tinyDelayMs,
-  unlockWallet,
-  withFixtures,
-} = require('../../helpers');
+const { strict: assert } = require('node:assert');
+const { PAGES } = require('../../webdriver/driver');
+const { withFixtures, tinyDelayMs, unlockWallet } = require('../../helpers');
 const FixtureBuilder = require('../../fixture-builder');
-const { mockServerJsonRpc } = require('../ppom/mocks/mock-server-json-rpc');
-
-const BALANCE_CHECKER = '0xb1f8e55c7f64d203c1400b9d8555d050f94adf39';
-const ENS_PUBLIC_RESOLVER = '0x4976fb03c32e5b8cfe2b6ccb31c09ba78ebaba41';
-const ENS_REGISTRY_WITH_FALLBACK = '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e';
-
-async function mockInfura(mockServer) {
-  await mockServerJsonRpc(mockServer, [
-    [
-      'eth_blockNumber',
-      {
-        methodResultVariant: 'custom',
-      },
-    ],
-    ['eth_estimateGas'],
-    ['eth_feeHistory'],
-    ['eth_gasPrice'],
-    ['eth_getBalance'],
-    ['eth_getBlockByNumber'],
-    ['eth_getTransactionCount'],
-    ['net_version'],
-  ]);
-
-  // Resolver Call
-  await mockServer
```

Comment:
Why are these mocks no longer needed?

Reply:
Because we removed the fixture for being on Mainnet below: `fixtures: new FixtureBuilder().withNetworkControllerOnMainnet().build(),`
it looks like this was not needed for the test